dataimportingfunctions.py

```diff
@@ -0,0 +1,26 @@
+import os
+from natsort import natsorted # Sort 1, 10, 2, 3 to 1, 2, 3, 10
+
+def get_data_dir_list(expnum, basedir):
+    """
+    get `.txt` data directory list for specified experiments
+    For example, when expnum=1, it returns ["MyData/trial-1", "MyData/trial-2", ..., "MyData/trial-n"]
+    """
+    
+    experimentlist = os.listdir(basedir)  # get a list of file names in `basedir`
+    experimentlist = [ex for ex in experimentlist if 'Experiment' in ex]  # only get `Experiment` folders
+    experimentlist = natsorted(experimentlist)  # `natsorted` enables to sort "1, 10, 2, ..." to "1, 2, ..., 10, ..."

```

Comment:
Merge these three lines together. the variable changes type so many times in this function :(